web-extensions/chrome/scripts/load-bookmarks.js

```diff
@@ -338,11 +389,11 @@ function addEvents(mangaList) {
         evData => evData.type === 'action' && evData.action === 'notification' && evData.operation === 'track',
         evData => {
             if (evData.error) {
+                // eslint-disable-next-line no-console
```

Comment:
Do we have at least an issue to track such linter-disabling lines and fix them by fixing the code itself ?
I understand not updating code you did not update, but it should be ideally updated along with the linter configuration change.
This should be folded into the commit updating the eslint configuration at least in the meantime.